Feliz/Styles.fs

```diff
@@ -89,410 +94,652 @@ type style =
-    static member inline flexBasis (value: ICssUnit) = Interop.mkStyle "flexBasis" value
-    static member inline flexGrow (value: int) = Interop.mkStyle "flexGrow" value
+    /// Sets the margin area on all four sides of an element. It is a shorthand for margin-top, margin-right, 
+    /// margin-bottom, and margin-left.
     static member inline margin(value: ICssUnit) = Interop.mkStyle "margin" value
+    /// Sets the margin area on the left side of an element. A positive value places it farther from its 
+    /// neighbors, while a negative value places it closer.
     static member inline marginLeft(value: int) = Interop.mkStyle "marginLeft" value
+    /// Sets the margin area on the left side of an element. A positive value places it farther from its 
+    /// neighbors, while a negative value places it closer.
     static member inline marginLeft(value: ICssUnit) = Interop.mkStyle "marginLeft" value
+    /// sets the margin area on the right side of an element. A positive value places it farther from its 
+    /// neighbors, while a negative value places it closer.
     static member inline marginRight(value: int) = Interop.mkStyle "marginRight" value
+    /// sets the margin area on the right side of an element. A positive value places it farther from its 
+    /// neighbors, while a negative value places it closer.
     static member inline marginRight(value: ICssUnit) = Interop.mkStyle "marginRight" value
+    /// Sets the margin area on the top of an element. A positive value places it farther from its 
+    /// neighbors, while a negative value places it closer.
     static member inline marginTop(value: int) = Interop.mkStyle "marginTop" value
+    /// Sets the margin area on the top of an element. A positive value places it farther from its 
+    /// neighbors, while a negative value places it closer.
     static member inline marginTop(value: ICssUnit) = Interop.mkStyle "marginTop" value
+    /// Sets the margin area on the bottom of an element. A positive value places it farther from its 
+    /// neighbors, while a negative value places it closer.
     static member inline marginBottom(value: int) = Interop.mkStyle "marginBottom" value
+    /// Sets the margin area on the bottom of an element. A positive value places it farther from its 
+    /// neighbors, while a negative value places it closer.
     static member inline marginBottom(value: ICssUnit) = Interop.mkStyle "marginBottom" value
+    /// Sets the padding area on all four sides of an element. It is a shorthand for padding-top, 
+    /// padding-right, padding-bottom, and padding-left.
+    static member inline padding(top: ICssUnit, right: int) =
+        Interop.mkStyle "padding" (
+            (unbox<string> top) + " " +
+            (unbox<string>right) + "px"
+        )
+    /// Sets the padding area on all four sides of an element. It is a shorthand for padding-top, 
+    /// padding-right, padding-bottom, and padding-left.
```

Comment:
Same as my comment for the margin, this overload specifically doesn't set the padding-left